projects/ng2-signalr/package.json

```diff
@@ -1,14 +1,16 @@
 {
-  "name": "ng2-signalr",
-  "version": "12.0.0",
+  "name": "@henkkelder/ng2-signalr",
+  "version": "18.0.1",
   "peerDependencies": {
-    "@angular/common": ">=11.0.0",
-    "@angular/core": ">=11.0.0"
+    "@angular/common": ">=18.0.0",
+    "@angular/core": ">=18.0.0",
```

Comment:
Hello @henkkelder 
Great job and my thanks for the update, however I have a question: Are you sure it should be 18+?
Because this package is not suitable for angular from 12 to 18 version

Reply:
Your question confuses me. I am using it myself with Angular 18. 
So what do you mean by 'not suitable for angular from 12 to 18'?

Reply:
Yes, you can use version 18, but for example I am currently working with 16 and your changes will not work for me, since the dependencies include version 18 of Angular. However, in the readme you wrote that the supported version is 12+

With this merge request you are not changing your library, but the global one, which is used by other users besides you and me.

Reply:
Ah you are right. Angular 18 introduced some breaking changes in Angular components that made it neccesary to update the dependencies.

I have just updated the readme to reflect this. 

Are you maintaining this repository now?